lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
BTW, I was wondering why the version still is 1.5.0 in master. Don't you merge back the release branches to master?